metadata.rb

```diff
@@ -6,7 +6,7 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 source_url 'https://github.com/cvent/octopus-deploy-cookbook'
 issues_url 'https://github.com/cvent/octopus-deploy-cookbook/issues'
-version '0.4.1'
+version '0.4.2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.